lib/RdfParser.ts

```diff
@@ -2,28 +2,13 @@ import { ActionContext, Actor } from "@comunica/core";
 import * as RDF from "@rdfjs/types";
 import {Readable, PassThrough} from "stream";
 import { MediatorRdfParseHandle, MediatorRdfParseMediaTypes } from '@comunica/bus-rdf-parse';
+import mediaMappings from "./mediaMappings";
```

Comment:
I guess this new file should also be added to the `files` array in package.json?